packages/application-shell/src/components/application-shell/application-shell.spec.js

```diff
@@ -866,7 +866,7 @@ describe('when clicking on navbar menu toggle', () => {
     });
 
     // Verify that the "Support" link is not rendered when the menu is collapsed
-    expect(navbarRendered.queryByText('Support')).toBeNull();
+    expect(navbarRendered.getAllByText('Support')).toHaveLength(1);
```

Comment:
Here, it is strange that the `display: none` property is supposed to take out the tooltip support from dom but it only just doesnt show it and its still in the dom

Reply:
Maybe there's a confusion about what `display: none` actually does.

It is not supposed to remove the element from the DOM but from the HTML rendering flow. This means it will not be included in the visible elements.
The main different between this and `visibility: hidden` is that this one hides the content but the content still uses its space (it's like making it transparent).

The only way for not including the element in the DOM is not to render it from the React component.

Reply:
This makes sense. Thank you 🙏🏾

Updated here: https://github.com/commercetools/merchant-center-application-kit/pull/3432/commits/955b9d531f31df690caa06d8ff239eed6b36f720